# Message for a lights topic

uint8 ON  = 1
uint8 OFF = 0

uint8  id_number	# Identify the object
string id_name		# Object name
uint8  state		# on or off
